diff --git a/ios/Core/Records/Record.swift b/ios/Core/Records/Record.swift
index f74c13d..a9eb5c0 100644
--- a/ios/Core/Records/Record.swift
+++ b/ios/Core/Records/Record.swift
@@ -130,4 +130,4 @@ extension Record {
-    try fieldsOf(self).forEach { field in
+    for field in fieldsOf(self) {
       guard let key = field.key else {
-        return
+        continue
       }
@@ -138,7 +138,7 @@ extension Record {
           if field.isRequired {
             try field.set(nil, appContext: appContext)
           }
-          return
+          continue
         }
         try field.set(jsValue: property, appContext: appContext)
       } else if field.isRequired {
